sdk/servicelinker/Azure.ResourceManager.ServiceLinker/api/Azure.ResourceManager.ServiceLinker.net8.0.cs

```diff
@@ -168,6 +467,205 @@ protected override void JsonModelWriteCore(System.Text.Json.Utf8JsonWriter write
+        public static bool operator ==(Azure.ResourceManager.ServiceLinker.Models.DeleteOrUpdateBehavior left, Azure.ResourceManager.ServiceLinker.Models.DeleteOrUpdateBehavior right) { throw null; }
+        public static implicit operator Azure.ResourceManager.ServiceLinker.Models.DeleteOrUpdateBehavior (string value) { throw null; }
+        public static bool operator !=(Azure.ResourceManager.ServiceLinker.Models.DeleteOrUpdateBehavior left, Azure.ResourceManager.ServiceLinker.Models.DeleteOrUpdateBehavior right) { throw null; }
+        public override string ToString() { throw null; }
+    }
+    public partial class DryrunOperationPreview : System.ClientModel.Primitives.IJsonModel<Azure.ResourceManager.ServiceLinker.Models.DryrunOperationPreview>, System.ClientModel.Primitives.IPersistableModel<Azure.ResourceManager.ServiceLinker.Models.DryrunOperationPreview>
+    {
+        internal DryrunOperationPreview() { }
+        public string Action { get { throw null; } }
+        public string Description { get { throw null; } }
+        public string Name { get { throw null; } }
+        public Azure.ResourceManager.ServiceLinker.Models.DryrunPreviewOperationType? OperationType { get { throw null; } }
+        public string Scope { get { throw null; } }
+        protected virtual void JsonModelWriteCore(System.Text.Json.Utf8JsonWriter writer, System.ClientModel.Primitives.ModelReaderWriterOptions options) { }
+        Azure.ResourceManager.ServiceLinker.Models.DryrunOperationPreview System.ClientModel.Primitives.IJsonModel<Azure.ResourceManager.ServiceLinker.Models.DryrunOperationPreview>.Create(ref System.Text.Json.Utf8JsonReader reader, System.ClientModel.Primitives.ModelReaderWriterOptions options) { throw null; }
+        void System.ClientModel.Primitives.IJsonModel<Azure.ResourceManager.ServiceLinker.Models.DryrunOperationPreview>.Write(System.Text.Json.Utf8JsonWriter writer, System.ClientModel.Primitives.ModelReaderWriterOptions options) { }
+        Azure.ResourceManager.ServiceLinker.Models.DryrunOperationPreview System.ClientModel.Primitives.IPersistableModel<Azure.ResourceManager.ServiceLinker.Models.DryrunOperationPreview>.Create(System.BinaryData data, System.ClientModel.Primitives.ModelReaderWriterOptions options) { throw null; }
+        string System.ClientModel.Primitives.IPersistableModel<Azure.ResourceManager.ServiceLinker.Models.DryrunOperationPreview>.GetFormatFromOptions(System.ClientModel.Primitives.ModelReaderWriterOptions options) { throw null; }
+        System.BinaryData System.ClientModel.Primitives.IPersistableModel<Azure.ResourceManager.ServiceLinker.Models.DryrunOperationPreview>.Write(System.ClientModel.Primitives.ModelReaderWriterOptions options) { throw null; }
+    }
+    public partial class DryrunPatch : System.ClientModel.Primitives.IJsonModel<Azure.ResourceManager.ServiceLinker.Models.DryrunPatch>, System.ClientModel.Primitives.IPersistableModel<Azure.ResourceManager.ServiceLinker.Models.DryrunPatch>
```

Comment:
this name should follow the same change for the name change for `Dryrun` model above